views/spaces.erb

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
We found a nice HTML skeleton we can use across the views, includes a link to a stylesheet which we can use later